docs/core/tools/dotnet-install-script.md

```diff
@@ -63,13 +60,13 @@ The default value is `LTS`. For more information on .NET support channels, see t
 
 `-Version <VERSION>`
 
-Represents a build version on the source channel (see the `-Channel` option). The possible values are:
+Represents a specific build version. The possible values are:
 
-- `latest` - Latest build on the channel
-- `coherent` - Latest coherent build on the channel; uses the latest stable package combination
-- Three-part version in X.Y.Z format representing a specific build version (for example, `1.0.x` with `x` representing the patch version; or a specific build, such as `2.0.0-preview2-006120`)
+- `latest` - Latest build on the channel (used with the `-Channel` option)
+- `coherent` - Latest coherent build on the channel; uses the latest stable package combination (used with Branch name `-Channel` options)
+- Three-part version in X.Y.Z format representing a specific build version; supercedes the `-Channel` option (for example, a specific build, such as `2.0.0-preview2-006120`)
```

Comment:
; supercedes -> . It supersedes 

(for example, a specific build, such as `2.0.0-preview2-006120`) -> For example, `2.0.0-preview2-006120`.

Reply:
done...

Reply:
done...